### PR TITLE
update the service mesh icon with patterns icon

### PR DESCRIPTION
**Pull request by Muhu-Muza**
**Description**

This PR fixes [#1744](https://github.com/meshery/meshery.io/issues/1744)

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by Muhu-Muza** [comment]
hello @vishalvivekm ,
This PR reflects  the  changes, kindly  regard  this  as  it is  more  concise

File: _includes/modal.html [hunk 1]
```diff
@@ -27,9 +27,9 @@ <h6 class="pattern-type">{{pattern.type | upcase }}</h6>
       {%else%}
       <img
         class="image-class"
-        data-snapshot-light="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-light.png"
-        data-snapshot-dark="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
-        src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
+        data-snapshot-light="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
+        data-snapshot-dark="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
+        src="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
```
Comment (vishalvivekm) [inline _includes/modal.html:32]:
```suggestion
        src="https://raw.githubusercontent.com/meshery/meshery.io/master/assets/images/logos/service-mesh-pattern.svg"
```

Comment (vishalvivekm) [reply _includes/modal.html:32]: @Muhu-Muza just this one change is required.

Comment (vishalvivekm) [inline _includes/modal.html:32]:
```suggestion
     data-snapshot-light="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-light.png"
        data-snapshot-dark="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
        src="https://raw.githubusercontent.com/layer5labs/meshery-extensions-packages/master/action-assets/design-assets/{{pattern.patternId}}-dark.png"
```

Comment (vishalvivekm) [reply _includes/modal.html:32]: Let's revert these changes, as we just need to change the fallback svg not the dark / light theme svgs.

**Comment by vishalvivekm** [comment]
Hi, @Muhu-Muza 
looks like the issue was addressed in a month old pull request, which was recently merged: #1705 

So, I'd consider the issue as completed, as meshery.io/catalog modal service mesh icons are updated with pattern's.

Thank you for your contribution

**Pull request closed without merging**